### ISSUE TITLE
Failed to run example 00_Initialization on Linux

### ISSUE THREAD

**Issue by define-private-public** [zacharycarter/zengine#2]
debug output:
```
INFO [TEX ID 1] Texture created successfully (1x1)
INFO [TEX ID 1] Base white texture loaded successfully
DEBUG [VSHDR ID 1] Vertex shader compiled successfully
DEBUG [VSHDR ID 2] Fragment shader compiled successfully
DEBUG [SHDR ID 3] Shader program loaded successfully
INFO [VAO ID 1] Default buffers VAO initialized successfully (lines)
INFO [VAO ID 2] Default buffers VAO initialized successfully (triangles)
Traceback (most recent call last)
main.nim(152)            main
core.nim(46)             init
zgl.nim(483)             zglInit
zgl.nim(414)             loadDefaultBuffers
opengl.nim(1577)         glBufferData
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
```

I'm running Ubuntu 16.04.x.  zengine version is `0094b4ebcfe2105ce57937d766d8adbb93db7366`.

**Comment by define-private-public** [comment]
Nevermind, I didn't realize that I need to run the examples from the root directory.  Put that in the Readme?

**Comment by zacharycarter** [comment]
Will do

**Comment by zacharycarter** [comment]
Closing - addressed by : 2d4c78e07753a8698dc25e5aac2f9b29b683bb62
